core/src/tpu.rs

```diff
@@ -148,6 +148,9 @@ impl Tpu {
 
         let (non_vote_sender, non_vote_receiver) = banking_tracer.create_channel_non_vote();
 
+        // 25K PPS
+        const MAX_STREAMS_PER_MS_TPU: u64 = 25;
```

Comment:
How did we come up with this constant? Did we do any testing on the perf impact?

Reply:
It was used in our testing on `3gG` -- I tested using bench-tps -- no observable difference probably because it is still well above our capability to hit with current bench-tps tool. But I revert this change to later keep using the current values.